packages/runtime/src/index.ts

```diff
@@ -10,7 +10,8 @@ import { assert } from './utils/logger';
 
 export { FederationHost } from './core';
 export { registerGlobalPlugins } from './global';
-export { loadScript } from './utils';
+export type { Federation } from './global';
+export type { FederationRuntimePlugin } from './types';
```

Comment:
i have export it , and you can use import  @module-federation/runtime/type to get

Reply:
done